web/console/src/app/components/common/ModalMarketplace/index.tsx

```diff
@@ -0,0 +1,73 @@
+// Copyright (C) 2021 Creditor Corp. Group.
+// See LICENSE for copying information.
+
+import { Dispatch, SetStateAction } from 'react';
+
+import { Lot } from '@/marketplace';
+import { PlayerCard } from '../PlayerCard';
+
+import CloseModal from '@/app/static/img/MarketPlacePage/marketplaceModal/close.svg';
+
+import './index.scss';
+
+const ONE_COIN = 1;
+
+export const ModalMarketPlace: React.FC<{ lot: Lot; setShowModal: Dispatch<SetStateAction<boolean>> }> =
+    ({ lot, setShowModal }) => {
+        /** TODO: add function entity */
+        const buyNowButton = () => { };
+        const bidButton = () => { };
+
+        return <div className="marketplace-modal">
+            <div className="marketplace-modal__wrapper">
+                <div>
+                    <button onClick={() => setShowModal(false)} className="marketplace-modal__close-button">
+                        <img src={CloseModal} alt="modal close button" className="marketplace-modal__close-button__img" />
+                    </button>
+                </div>
+                <h2 className="marketplace-modal__title">{lot?.card.playerName}</h2>
```

Comment:
what will be shown if there is no lot?
undefined?
Do we need to add some check?
Like this: 
_lot && lot.card.playerName ? lot?.card.playerName : 'not lot'_
_PS. Just double check this_

Reply:
It was written by mistake, the lot will exist in all cases